modules/machineset-gcp-confidential-vm.adoc

```diff
@@ -49,11 +41,35 @@ spec:
           confidentialCompute: Enabled <1>
           onHostMaintenance: Terminate <2>
           machineType: n2d-standard-8 <3>
+endif::cpmso[]
+ifdef::cpmso[]
+apiVersion: machine.openshift.io/v1
+kind: ControlPlaneMachineSet
+...
+    machines_v1beta1_machine_openshift_io:
+      spec:
+        providerSpec:
+          value:
+            confidentialCompute: Enabled <1>
+            onHostMaintenance: Terminate <2>
+            machineType: n2d-standard-8 <3>
+endif::cpmso[]
 ...
 ----
-<1> Specify whether Confidential VM is enabled. Valid values are `Disabled` or `Enabled`.
+<1> Specify whether Confidential VM is enabled. Valid values are:
+
+`Enabled`:: Enables Confidential VM with a default selection of Confidential VM technology.
```

Comment:
Do we know what this is? Seems users might want to know.

Reply:
The default is AMD SEV per https://github.com/openshift/cluster-api-provider-gcp/blob/release-4.19/api/v1beta1/gcpmachine_types.go#L350C5-L350C166